### ISSUE TITLE
no wrap issue with chrome

### ISSUE THREAD

**Issue by jtaleric** [redhat-performance/ops-tools#1]
Please fix the no-wrap issues when viewed with chrome. The table layout becomes hard to follow.

**Comment by sadsfae** [comment]
We're currently using this CSS per W3C standards to ensure this doesn't happen

```
word-wrap: normal;
word-break: keep-all;
```

Apparently this isn't good enough for some browsers (renders fine for me on Firefox minus the r730xd systems).  Let's track this and dig into why it's not being reflected on some browsers like Chrome.  Curious, how does it look on Firefox?

**Comment by sadsfae** [comment]
This looks ok now in both Firefox and Chrome, please confirm you see the same.  This seems to be a [bug](https://kenneth.io/blog/2012/03/04/word-wrapping-hypernation-using-css/) in Chrome where it ignores the `word-break: keep-all;` CSS directive
